exemplar-crd/tools.go

```diff
@@ -0,0 +1,23 @@
+// +build tools
+
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1alpha1
+
+import (
+	_ "sigs.k8s.io/controller-tools/pkg/version"
```

Comment:
Why not just include this in the file that already exists?

Reply:
while we could do that, it's not inherently a part of the other source files. Moving under a `hack` directory.